tests/opentrace/test_settings.py

```diff
@@ -0,0 +1,37 @@
+import opentracing
+import pytest
+import unittest
+
+from ddtrace.settings import ConfigException
+from ddtrace.opentrace import Config
+
+
+class OpenTraceConfigTest(unittest.TestCase):
+    """Test for the opentrace config"""
+
+    open_tracer = None
```

Comment:
I'd say to not use / store objects. The good thing of `pytest` is that we can use powerful fixtures that can inject / change global states temporarily. You should create (in another file) the Config fixture with a setup / teardown steps: https://docs.pytest.org/en/latest/fixture.html#fixture-finalization-executing-teardown-code